### PR TITLE
Homework is done!

### PR DESCRIPTION
**Pull request by BillyWoods**

### CODE REVIEW

File: timetracker/timetracker/urls.py [hunk 1]
```diff
@@ -17,17 +17,11 @@
 from django.contrib import admin
 from django.http import HttpResponse
```
Comment (MarkusH) [inline timetracker/timetracker/urls.py:18]: You don't need this import anymore.

File: timetracker/entries/views.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
 from django.shortcuts import render
+from django.http import HttpResponse
```
Comment (MarkusH) [inline timetracker/entries/views.py:2]: This import is stale, too.

File: timetracker/timetracker/urls.py [hunk 3]
```diff
@@ -17,17 +17,11 @@
...
-
-def hello_world(request):
-    name = request.GET.get('name')
-    if name:
-        return HttpResponse('Hello %s' % name)
-    else:
-        return HttpResponse('<form><input name="name"></form>')
-
-
+import entries.views
```
Comment (MarkusH) [inline timetracker/timetracker/urls.py:20]: I recommend to stay consistent with relative and absolute imports of your own code. In other words, in `timetracker/entries/views.py` you use `from .models import ...`. You can do that here too: `from .views import clients, home, projects`.

**Comment by MarkusH** [comment]
Looks good overall. :+1:

File: timetracker/timetracker/urls.py [hunk 4]
```diff
@@ -17,17 +17,11 @@
...
 
 urlpatterns = [
-    url(r'^$', hello_world),
+    url(r'^$', entries.views.home, name='homepage'), 
+	url(r'^clients/$', entries.views.clients, name='entries-viewClients'),
```
Comment (funkybob) [inline timetracker/timetracker/urls.py:24]: Try to keep your indenting consistent -- use only spaces, or only tabs.

In general, everyone recommends 4-space indenting, _never_ tabs.  A space is always a space.  A tab may be shown as 2, 3, 4, 8, or any number of spaces, but Python will always consider it 8 spaces.  If you mix tabs and spaces, you may get "inconsistent indenting" errors and not understand why.

**Comment by sesh** [comment]
@BillyWoods Just a heads up – you forked the lesson-four repository a little early (you can fork it again now and it has the homework ready to complete from the class).

**Pull request closed without merging**